packages/xo-server-test/src/vm/index.spec.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env jest */
+
+import { mapValues } from 'lodash'
+
+import config from '../_config'
+import xo from '../_xoConnection'
+
+describe('vm', () => {
+  describe('.set() :', () => {
+    const PROPERTIES_TO_SET_TO_NULL = {
+      coresPerSocket: 1,
+      cpuCap: 1,
+    }
```

Comment:
IMHO, this test is too complex, too magic, please keep things as simple as possible :slightly_smiling_face:

Reply:
My first idea is to let the test generic. If we want to test a properly we'll simply add it in this objects. But you're right is more complex.

Reply:
I think tests should be kept as simple and as literal as possible, otherwise we risk introducing bugs in the tests themselves.